test/test_infer_util.py

```diff
@@ -130,14 +131,21 @@ def test_prior_predictive():
     assert predictive_samples["obs"].shape == (100,) + data.shape
 
 
-def test_log_likelihood():
+@pytest.mark.parametrize('batch_shape', [(), (100,), (2, 50)])
+def test_log_likelihood(batch_shape):
     model, data, _ = beta_bernoulli()
-    samples = Predictive(model, return_sites=["beta"], num_samples=100)(random.PRNGKey(1))
-    loglik = log_likelihood(model, samples, data)
+    samples = Predictive(model, return_sites=["beta"], num_samples=200)(random.PRNGKey(1))
```

Comment:
What will happen if we provide `batch_ndims=2` here?

Reply:
I believe the output will have shape `(1, 200, ...)` but let me test it.

Reply:
That's true, I just add a test for that in `test_prior_predictive`.